src/passes/MemoryCopyFillLowering.cpp

```diff
@@ -32,12 +32,26 @@ struct MemoryCopyFillLowering
       return;
     }
     if (module->features.hasAtomics()) {
-      throw "Don't use this pass with atomics"; // alternatively... just don't unset bulkmem feature if atomics? or do nothing?
+      return;
     }
```

Comment:
I still think we should just remove these 3 lines.   Silently not lowering seems worse.  

I don't see how atomics has anything to do with memory.copy of memory.fill and I don't think its the job of this pass to second guess what emscripten whats to do.   If I want to lower my copy/fill operations I should be able to do it, even if I have some atomics in my code.